bot-twitter/src/keeper.ts

```diff
@@ -0,0 +1,53 @@
+import { AuctionInitialInfo } from 'auctions-core/src/types';
+import getSigner from 'auctions-core/src/signer';
+import { enrichAuction } from 'auctions-core/src/auctions';
+import {
+    authorizeCollateral,
+    authorizeWallet,
+    getCollateralAuthorizationStatus,
+    getWalletAuthorizationStatus,
+} from 'auctions-core/src/authorizations';
+
+const ETHEREUM_NETWORK = process.env.ETHEREUM_NETWORK || 'kovan';
```

Comment:
You should not hardcode default option several times (see `index.ts:8`), instead, you may want to create a `variables.ts` file where you do this once and export